### PR TITLE
New package: mup-6.9.

### PR DESCRIPTION
**Pull request by fosslinux**

### CODE REVIEW

**Comment by fosslinux** [comment]
Ping

File: srcpkgs/mup/template [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Template file for 'mup'
+pkgname=mup
+version=6.9
+revision=1
+build_style=gnu-configure
+# Multiple things are broken with multithreaded compile
+make_build_args="-j1"
```
Comment (ericonr) [inline srcpkgs/mup/template:7]: `disable_parallel_builds`?

**Review by ericonr — COMMENTED** [review]
The cross hacking looks somewhat bothersome to maintain when new updates come out. Maybe this is best as a `nocross` package? Not sure.

File: srcpkgs/mup/template (hunk 1 above)
Comment (fosslinux) [reply srcpkgs/mup/template:7]: Oh yeah that exists now. The cross hacking I am indifferent on. From 6.3 -> 6.9 I didn't need to change anything but I don't have a strong preference either way.

**Comment by fosslinux** [comment]
I swear these "Transient resolver failures"

File: srcpkgs/mup/template [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+pkgname=mup
+version=6.9
+revision=1
+build_style=gnu-configure
+hostmakedepends="ghostscript groff flex bison netpbm automake"
+makedepends="fltk-devel libXpm-devel"
+short_desc="Free, yet very feature rich, music publication program"
+maintainer="fosslinux <fosslinux@aussies.space>"
+license="BSD-3-Clause"
+homepage="http://www.arkkra.com/"
```
[the remark this answers is not in the document]
Comment (fosslinux) [reply srcpkgs/mup/template:11]: That's odd.. I'm able to;

> It is now available in Version 6.9, which was released in 2021

**Pull request closed without merging**